index.html

```diff
@@ -100,16 +100,16 @@ <h3 class="kva-section-summary">For Developers</h3>
         <img src="images/developer.jpg" class="kva-people img-fluid">
       </div>
       <div class="col-md-6">
-        <p class="kva-people-title">On-chain data, off-chain application logic</p>
+        <p class="kva-people-title">Immutable On-Chain Data</p>
         <p class="kva-people-text">
-          It is never so easy to develop a dApp! Use your favorite programming language, easy to deploy, easy to update.
-          The data stays on blockchain, truely decentralized.
+          A simple solution to a critical and fast growing requirement. Especially suited to NFT and Metaverse Metadata.
+          Inherently Decentralized & Incentivised by Low Cost Low Power PoW Mining.
```

Comment:
Simplify a bit? "Incentivised" is a bit too advanced for non-English speaker?

Reply:
Mining a bit redundant since has its own area under "Why Kevacoin?" maybe just remove its mention here and cover incentives better under FAQ?:

A simple solution to a critical and fast growing requirement that is inherently decentralized. Especially suited to NFT and Metaverse Metadata.

Reply:
I agree, let's remove mining here and use this sentence as you suggested:

> A simple solution to a critical and fast growing requirement that is inherently decentralized. Especially suited to NFT and Metaverse Metadata.